screens/lesson_elements/ImageCaptureLessonScreen.tsx

```diff
@@ -48,11 +54,15 @@ export default function ImageCaptureLessonScreen({
 
   const [imageCaptured, setImageCaptured] = useState<Boolean>(false);
   const [showNavigation, setShowNavigation] = useState<boolean>(true);
+  const [showChatBubble, setShowChatBubble] = useState<boolean>(false);
```

Comment:
nit: this area can show multiple bubbles, so `showChats` or `showChatArea` would be a little clearer